### PR TITLE
Adding vtkplotter

### PR DESCRIPTION
**Pull request by RubendeBruin**
This is the conda-forge recipe for vtkplotter.

Checklist:
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged. (N/A)
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details) : frompypi
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there:  TODO

### CODE REVIEW

**Comment by RubendeBruin** [comment]
@conda-forge/help-python 

I have a problem passing the tests on linux_64.
The vtkplotter package can not run headless without installing some additional packages/settings on the host machine. etc. How can we solve this?

This is a pure-python package, it passes on osx and windows, it would have my preference to allow it to fail tests on linux.

**Comment by marcomusy** [comment]
I confirm that I am willing to act as maintainer of the recipe.

File: recipes/vtkplotter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d9db1c6f1e267ef672cf7721cdc270057d8e22428b8aa04e1e0a2b119b1d28e9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/vtkplotter/meta.yaml]: can this be noarch python?

Comment (RubendeBruin) [reply recipes/vtkplotter/meta.yaml]: Yup, source and data only so I guess that is a "yes". Implemented and pushed

**Comment by RubendeBruin** [comment]
@conda-forge/help-python Ready for review

**Pull request merged**